### PR TITLE
feat(): reviewType and new prompt construction method

### LINKED ISSUE #80: [NPM Packaging] Use Risk level instead of LOGAF
Description:
Some formatting with the comment to make it look nicer.

No one knows what LOGAF means. Use Risk and rate 1 low -> 5 high.
Also add Review to the sign off

### PR DESCRIPTION
**Pull request by mattzcarey**
also fixes json bug

closes #80 #79

### CODE REVIEW

File: src/common/utils/parseAndDecodeAttributes.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  processor: (value: string) => string
+) => {
+  attributesToEncode.forEach((attribute) => {
+    if (object[attribute]) {
+      object[attribute] = processor(object[attribute]);
+    }
+  });
+};
+
+export const parseAndDecodeAttributes = <T>(
```
Comment (gowoons) [inline src/common/utils/parseAndDecodeAttributes.ts:28]: It could be good to rename this to `parseAttributes`

**Review by lizacullis — APPROVED** [review]
This is some seriously cool stuff! Very nice!

File: src/test/cases/snapshots/too-many-nested-loops.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-**LOGAF Level 1 - src/test/cases/.cache/5519e4e1b45143a504ec259a5d911dea930372c19b3f56b51afab53f55339b56.ts**
+**LOGAF Level 4 - src/test/cases/.cache/5519e4e1b45143a504ec259a5d911dea930372c19b3f56b51afab53f55339b56.ts**
```
Comment (gowoons) [inline src/test/cases/snapshots/too-many-nested-loops.md:1]:
```suggestion
**Risk Level 4 - src/test/cases/.cache/5519e4e1b45143a504ec259a5d911dea930372c19b3f56b51afab53f55339b56.ts**
```

File: src/test/cases/snapshots/unawaited-promise.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-**LOGAF Level 2 - src/test/cases/.cache/eb303e5498352ed36e5d1d2d4a09e085574ec3d2da6564ea0e7ea6caabec7c0f.ts**
+**LOGAF Level 3 - src/test/cases/.cache/eb303e5498352ed36e5d1d2d4a09e085574ec3d2da6564ea0e7ea6caabec7c0f.ts**
```
Comment (gowoons) [inline src/test/cases/snapshots/unawaited-promise.md:1]:
```suggestion
**Risk Level 3 - src/test/cases/.cache/eb303e5498352ed36e5d1d2d4a09e085574ec3d2da6564ea0e7ea6caabec7c0f.ts**
```

File: src/common/utils/parseAndDecodeAttributes.ts
[the remark this answers is not in the document]
Comment (mattzcarey) [reply src/common/utils/parseAndDecodeAttributes.ts:1]: Yep!!

**Pull request merged**